components/journal/Meal.tsx

```diff
@@ -72,23 +78,32 @@ const Meal = ({ food, id, type, logId }: MealProp) => {
               </ul>
             </div>
             <div className="flex gap-3 justify-between">
-              <Button className="p-3 w-28 flex gap-2" variant={'outline'} onClick={handleEditButton}>
+              <Button
+                className="p-3 w-28 flex gap-2"
+                variant={'outline'}
+                onClick={handleEditButton}
+              >
                 <Pencil width={17} />
                 Edit
               </Button>
               <AlertDialog>
                 <AlertDialogTrigger>
-                  <Button className="p-3 w-28 flex gap-2" variant={'destructive'}>
+                  <Button
+                    className="p-3 w-28 flex gap-2"
+                    variant={'destructive'}
+                  >
                     <Trash2 width={17} />
                     Delete
```

Comment:
**Warning: validateDOMNesting(...): <button> cannot appear as a descendant of <button>** appears on clicking collapsible from this part of the code. When clicking on the Meal component - the collapsible shadcn window this error appears. It's not killing application but I don't think it should be there. I think this confirmation window "Are you sure you want to delete this meal?" causes the issue but I don't understand why. @johnie